apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.js

```diff
@@ -152,6 +152,12 @@ window.PLOrderBlocks = function (uuid, options) {
       let leftDiff = calculateIndent(ui, placeholder.parent());
       placeholder[0].style.marginLeft = leftDiff + 'px';
       placeholder[0].style.height = ui.item[0].style.height;
+
+      //enable width if the dropzone isn't bottom
+      if (options.dropzoneLayout === 'pl-order-blocks-bottom'){
+        placeholder[0].style.width = ui.item[0].style.width;
+      }
+
```

Comment:
This shouldn't be necessary once you fix the styles as I said above.

Reply:
I think this is still needed since it's setting the placeholder width. When removed and with the previously mentioned css changes in place I get this: 
![image](https://github.com/PrairieLearn/PrairieLearn/assets/16738261/2b03589d-1848-4a75-8936-51e1fe51d494)

Reply:
Oh, sorry, I got confused about what this was doing. Shouldn't we be doing this based on if `inline=true`, not based on where the dropzone is?

You can pass the `inline` boolean into the `PLOrderBlocks` creation function call as a property on the `options` object. Then this would just be `if (options.inline) {....`